src/model/neural_network/neural_network.py

```diff
@@ -0,0 +1,104 @@
+from typing import List
+
+import numpy as np
+
+from model.neural_network.activation_function_utils import (
+    activation_functions_utils,
+    ActivationFunctionUtils,
+)
+
+np.set_printoptions(suppress=True)
+
+
+class NeuralNetworkHiddenLayerInfo:
+    def __init__(self, activation_function_name: str, neurons_count: int):
+        self.activation_function_name: str = activation_function_name
+        self.neurons_count: int = neurons_count
+
+
+class NeuralNetworkHiddenLayer:
+    def __init__(self, basic_info: NeuralNetworkHiddenLayerInfo, weights: np.ndarray, biases: np.ndarray):
+        self.info = basic_info
+        self.utils: ActivationFunctionUtils = activation_functions_utils[
+            basic_info.activation_function_name
+        ]
+        self.weights: np.ndarray = weights
+        self.biases = biases
+        self.values: np.ndarray = np.zeros(shape=(self.info.neurons_count,))
```

Comment:
`values` is not the state of a layer, but just its output for the given data, so it should not be an attribute.

Reply:
Done